src/shared/models/form.model.ts

```diff
@@ -37,3 +38,8 @@ export interface StacksTransactionFormValues {
   feeType: string;
   nonce?: number | string;
 }
+
+export interface BitcoinRecipient {
```

Comment:
There's nothing bitcoin specific about this. How could we name differently?

`TransferRecipient`?